src/Models/ModelInstantiator.php

```diff
@@ -353,7 +353,9 @@ public function convertPropertyOnSave(
             return $property;
         }
 
-        return $column->getConverter()->onSave($property);
+        return $property === null
```

Comment:
Minor style note: This is terse and readable, so no change is needed, but my instinct would be to have the null check as a separate conditional with a commented explanation (e.g. "Null values are not converted... etc"), like the conditional on line 351.

Anyway, I don't necessarily see these methods growing with age, so it's fine to be concise.